mediawiki-extension.json

```diff
@@ -6,7 +6,7 @@
 	],
 	"url": "https://github.com/wmde/WikibaseDataModelSerialization",
 	"description": "Serializers and deserializers for the Wikibase DataModel",
-	"version": "2.3.0",
+	"version": "3.0.0",
```

Comment:
I would rather delete this line at all.
> Packagist uses VCS repositories, so the statement above is very much true for Packagist as well. Specifying the version yourself will most likely end up creating problems at some point due to human error.
(From: https://getcomposer.org/doc/04-schema.md#version)